### PR TITLE
New logo for readme

### LINKED ISSUE #394: Decide and implement a logo for Mitiq
Issue Description
-----------------
Mitiq doesn't have a logo.

Proposed Solution:
-----------------

- [ ] Propose some alternative logos
- [ ] Get feedback and maybe get votes from other developers 
- [ ] Implement the logo in the Readme.md and in the docs.

### PR DESCRIPTION
**Pull request by crazy4pi314**
Description
-----------
Adding new logo by @francespoblete to the readme! 
Co-authored-by:  Frances Joy Poblete<FrancesJoy.Poblete@haw-hamburg.de>
Closes #394 

Checklist
-----------

- [x] I added unit tests for new code.
- [x] I used [type hints](https://www.python.org/dev/peps/pep-0484/) in function signatures.
- [x] I used [Google-style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) docstrings for functions.
- [x] I [updated the documentation](../blob/master/docs/CONTRIBUTING_DOCS.md) where relevant.

### CODE REVIEW

**Comment by crazy4pi314** [comment]
@rmlarose @andreamari is the doc building broke on master rn? I just merged that in here and its failing tests.

**Comment by andreamari** [comment]
> @rmlarose @andreamari is the doc building broke on master rn? I just merged that in here and its failing tests.
I think sphinx doesn't like having second-level sections (`## Installation`) without having a top-level title  (`# Mitiq`)

**Comment by nathanshammah** [comment]
I think it's also better to keep the markdown title as it is, and just add the logo.

**Comment by nathanshammah** [comment]
It is still failing. This issue can be related: 
https://github.com/executablebooks/MyST-NB/issues/230

**Comment by nathanshammah** [comment]
Actually maybe it's just a sphinx thing, e..g, https://stackoverflow.com/questions/52138336/sphinx-reference-to-an-image-from-different-locations

File: docs/source/conf.py [hunk 1]
```diff
@@ -111,6 +111,7 @@
     "dollarmath",
     "html_image",
     "smartquotes",
+    "html_image"
```
Comment (rmlarose) [inline docs/source/conf.py:114]: I don't think we need this anymore since it was changed to markdown image format?

Comment (rmlarose) [reply docs/source/conf.py:114]: oh, also it's duplicated from above

Comment (rmlarose) [inline docs/source/conf.py:114]:
```suggestion
```

**Review by rmlarose — CHANGES_REQUESTED** [review]
We now have docs/assets and docs/source/img for images. Better to consolidate in my opinion.

**Comment by rmlarose** [comment]
> We now have docs/assets and docs/source/img for images. Better to consolidate in my opinion.

I'll merge now and do a follow-up PR for this.

test / test is failing because coverage can't be uploaded because who the hell knows why.

**Review by rmlarose — APPROVED** [review]

**Pull request merged**